core/src/main/scala/cats/data/Const.scala

```diff
@@ -35,7 +35,7 @@ final case class Const[A, B](getConst: A) {
     s"Const(${A.show(getConst)})"
 }
 
-object Const extends ConstInstances {
+object Const extends ConstInstances with ConstInstancesBinCompat0 {
```

Comment:
This is causing a Mima issue... Had it as a method before but of course that makes it a ReverseMissingMethod issue. What's our practice on binary incompatible instances for data types?

Reply:
I think this will work if you just create a new ` ConstInstances2` and let ` ConstInstances1` extend from it. IIRC MiMa is fine with it when using sealed abstract classes. :)